iwyu_path_util.cc

```diff
@@ -173,7 +176,11 @@ string MakeAbsolutePath(const string& base_path, const string& relative_path) {
 
 string GetParentPath(const string& path) {
   llvm::StringRef parent = llvm::sys::path::parent_path(path);
-  return parent.str();
+  string result = parent.str();
+  // Ensure trailing slash
+  if(!result.empty() && result[result.size() - 1] != '/')
```

Comment:
And you might need to consider Windows path separators here, or use `CanonicalizeFilePath` to turn the slashes the right way.